terraform/modules/cleanup-lambda/iam.tf

```diff
@@ -6,6 +6,7 @@ resource "aws_iam_role" "cleanup_lambda_role" {
 
 data "aws_iam_policy_document" "assume_role_cleanup_lambda" {
   statement {
+    sid     = "AllowAssumeRole"
```

Comment:
This file has 3 other SIDs beginning 'cleanupLambda', we should be consistent with our naming.